client/components/checklist/task.js

```diff
@@ -147,7 +147,7 @@ class Task extends PureComponent {
 		const isExpandable = ! inProgress && ( ! completed || completedButtonText );
 		const taskActionButtonText = completed
 			? completedButtonText
-			: buttonText || translate( 'Do it!' );
+			: buttonText || translate( 'Try it' );
```

Comment:
This change has the potential to affect _all_ the checklists.

- Simple sites
- Store (WooCommerce)
- Jetpack

I just want to make sure we're aware of the scope of this change 🙂

Reply:
yeah, thanks for bringing it up. I should have added the screenshots but I did take a look. This is one of those shared pieces of feedback that applies to all our products.